modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -306,6 +327,34 @@ And owners with:
 
 ----
 chown postgres:postgres /var/lib/containers/storage/volumes/var-pgsql
-chown wwwrun:www /var/lib/containers/storage/volumes/var-spacewalk
+chown wwwrun:www        /var/lib/containers/storage/volumes/var-spacewalk
 ----
```

Comment:
This will definitely not work from the container host.

Reply:
ok, removed it now.